### DISCUSSION TITLE
Use Frequency-division multiplexing (FDM) or Code-division Multiplexing (CDM)

### DISCUSSION THREAD

**Discussion by abdallah1097** [NVlabs/sionna#342] [General]
It is not practical for all users in the sector to send over all sub-carriers in the grid.

Isn't there a method to implement FDM where each user gets assigned a range of sub-carries to send over?
Isn't there a method to implement CDM where all users send over all sub-carriers, but each user gets assigned a code?

**Comment by SebastianCa** [comment]
Hi,
as of now, Sionna does not include a dedicated module for FDM or CDM. However, implementing a custom FDM solution can be achieved relatively easily. You can do this by generating multiple ranges of subcarriers and stacking them together before the transmission process.
